tests/acceptance/features/apiContract/propfind.feature

```diff
@@ -0,0 +1,25 @@
+@api @skipOnOcV10
+Feature: Propfind test
+  check that the PROPFIND response contains all the relevant value
+
+  Background:
+    Given these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+      | Brian    |
```

Comment:
why is Brian created?
are you planning to add more test scenarios that use Brian?

Reply:
no that must be because of I copy-pasted the step